clover_admin.rb

```diff
@@ -182,22 +182,104 @@ def self.object_action(...)
     form_options(wrapper: :div)
 
     link = lambda do |obj|
+      return "" unless obj
       "<a href=\"/model/#{obj.class}/#{obj.ubid}\">#{Erubi.h(obj.name)}</a>"
     end
 
     show_html do |obj, column|
       case column
       when :name
         link.call(obj)
-      when :project, :location
+      when :project, :location, :vm_host
         link.call(obj.send(column))
       end
     end
 
+    model Account do
+      order Sequel.desc(Sequel[:accounts][:created_at])
+      eager [:identities]
```

Comment:
Better to use eager_graph here so the table is already joined:

```suggestion
      eager_graph [:identities]
```

However, note that this will result in search results using provider_names as a filter will only show the related provider_names, and not all provider names. If that is a problem, you may want to keep the current approach of using eager.